lib/js/src/protocol/enums/FrameType.js

```diff
@@ -34,7 +34,7 @@ import { Enum } from '../../util/Enum.js';
 
 /**
  * @typedef {Enum} FrameType
- * @property {Object} MAP
+ * @property {Object} _MAP
```

Comment:
why in some cases used `_MAP` and in other `MAP`, can't we use same approach for all cases?

Reply:
@o-mishch `_MAP` should be used since the property is considered private. There are some instances in the project where `MAP` is used, but they will be updated to be `_MAP` prior to an official release.